wg-structured-logging/charter.md

```diff
@@ -0,0 +1,70 @@
+# WG Structured Logging Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README]
+and uses the Roles and Organization Management outlined in [sig-governance].
+
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+[Kubernetes Charter README]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+
+## Scope
+
+Modernize logging in Kubernetes core components, allowing users to efficiently consume, process, store and analyse 
+information stored in logs.
+
+### In Scope
+
+- Define the standard for logging - propose libraries, interfaces, metadata schema
+- Reduce friction for using logging - reduce dependencies and performance overhead
+- Give more choice over logging - allow pluggable logging implementation
+- Ensure quality consistent logging - overview migration, create documentation, tooling and educate reviewers
+- Prevent regressions caused by logging - measure performance overhead and log volume changes
+
+For all of the above, we will focus on core Kubernetes components and addons.
+Other SIG subprojects/components (e.g. SIG Scheduling descheduler) are out of
+scope.
```

Comment:
Could we consider amending this charter to provide tools that out of scope projects could use to apply the last four points above to themselves?